.evergreen/orchestration/drivers_orchestration.py

```diff
@@ -198,6 +198,8 @@ def get_docker_cmd():
     docker = shutil.which("podman") or shutil.which("docker")
     if not docker:
         return None
+    if os.name == "nt":
+        docker = docker.replace(os.sep, "/")
```

Comment:
```suggestion
    docker = PureWindowsPath(docker).as_posix()
```
I think this should reliably convert any Windows `'\\'` to `'/'` regardless of the OS. Needs `from pathlib import PureWindowsPath`.